app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -88,6 +100,26 @@ class TabTrayView(
             TabsTouchHelper(tray.tabsAdapter).attachToRecyclerView(tray)
         }
 
+        tabTrayItemMenu = TabTrayItemMenu(view.context, { view.tab_layout.selectedTabPosition == 0 }) {
+            when (it) {
+                is TabTrayItemMenu.Item.ShareAllTabs -> interactor.onShareTabsClicked(
+                    view.tab_layout.selectedTabPosition == 1
```

Comment:
let's add a computed property to this view now that we're using it everywhere

`val isPrivateModeSelected: Boolean get() = view.tab_layout.selectedTabPosition == PRIVATE_TAB_ID`